src/lumigo_tracer/extension/extension.py

```diff
@@ -69,6 +75,7 @@ def _finish_previous_invocation(self, current_bandwidth: Optional[int]):
             started=int(self.start_time.timestamp() * 1000),
             requestId=self.request_id,
             networkBytesUsed=current_bandwidth - self.bandwidth,
-            cpuUsageTime=[s.dump() for s in self.sampler.get_samples()],
+            cpuUsageTime=[s.dump() for s in self.spu_sampler.get_samples()],
+            memoryUsageTime=[s.dump() for s in self.memory_sampler.get_samples()],
```

Comment:
```suggestion
            cpuUsageTime=[s.dump() for s in self.sampler.get_cpu_samples()],
            memoryUsageTime=[s.dump() for s in self.sampler.get_memory_samples()],
```
This is what I would have expect